modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -96,13 +100,13 @@
      * @param method the HTTP method
      * @param hasEntityBody weather request has an entity body
      */
-    public TargetRequest(TargetConfiguration targetConfiguration, HttpRoute route, URL url,
-                         String method, boolean hasEntityBody) {
+    public TargetRequest(TargetConfiguration targetConfiguration, 
+            HttpRoute route, URL url, String method, boolean hasEntityBody) {
         this(targetConfiguration, route, method, url, hasEntityBody);
     }
 
-    public TargetRequest(TargetConfiguration targetConfiguration, HttpRoute route, String method,
-                         URL url, boolean hasEntityBody) {
+    public TargetRequest(TargetConfiguration targetConfiguration, 
+            HttpRoute route, String method, URL url, boolean hasEntityBody) {
```

Comment:
Could you explain reason we have 2 public constructors?

Reply:
I am also wondering. I can't found the reason in commit history, and it is difficult to find out where it is used.